rhobot.go

```diff
@@ -14,66 +15,111 @@ func main() {
 
 	app := cli.NewApp()
 	app.Name = "Rhobot"
-	app.Usage = "Rhobot is your friend."
+	app.Usage = "Rhobot is a database development tool that uses DevOps best practices."
 	app.EnableBashCompletion = true
 
+	config := config.NewConfig()
+
 	app.Commands = []cli.Command{
 		{
 			Name:    "run",
 			Aliases: []string{},
-			Usage:   "healthchecks|gocd|tbd",
+			Usage:   "healthchecks|pipeline|tbd",
 			Subcommands: []cli.Command{
 				{
 					Name:  "healthchecks",
-					Usage: "[database uri] [path to healthcheck file]",
+					Usage: "HEALTHCHECK_FILE [DATABASE_URI]",
 					Action: func(c *cli.Context) {
-						dburi := c.Args()[0]
-						path := c.Args()[1]
-						fmt.Println("DB_URI: ", dburi)
+
```

Comment:
The difference here is that I don't want to set the DBURI to "" if it is missing, I want to leave it as the default, or the value derived from environment variables.  So I could do this:

`if dbURI := c.Args().Get(1); if dbURI != "" {
  config.SetDBURI(dbURI)
}`

But I thought it was a wash in terms of which was more readable.  Same with the path, I would need to ensure it is not "", because it is a required argument.